_episodes/04-performance.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: "Teaching as a Performance Art"
+title: "Teaching as a Skilled Activity"
 teaching: 15
 exercises: 45
 questions:
 - "How do teachers improve?"
 objectives:
 - "Explain why standardized testing cannot by itself improve educational outcomes, and why and how peer-to-peer lesson study can."
 - "Explain how teaching practices are actually transferred and adopted."
-- "Summarize the strengths and weaknesses of your performance as a lecturer."
+- "Summarize the strengths and weaknesses of your teaching practices."
```

Comment:
This ties into the previous comment (about using the term "performance", and whether it needs to be emphasized).